src/insight/insights.service.ts

```diff
@@ -107,4 +107,22 @@ export class InsightsService {
 
     return new PageDto(entities, pageMetaDto);
   }
+
+  async findAllFeatured(pageOptionsDto: InsightPageOptionsDto): Promise<PageDto<DbInsight>> {
+    const queryBuilder = this.insightRepository.createQueryBuilder("insights");
+
+    queryBuilder
+      .andWhere("is_featured=true")
```

Comment:
You need a `where` before a `andWhere`:

```suggestion
    queryBuilder
      .where("is_featured=true")
      .andWhere("is_public=true")
```